ceph.spec.in

```diff
@@ -804,7 +804,6 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %post
```

Comment:
> You're right, we should add the systemd scriptlets to the ceph-osd/ceph-mon/ceph-mds packages

Especially since each daemon has its own systemd unit file, now.